docs/source/tutorials/chat.rst

```diff
@@ -98,16 +98,16 @@ prompt:
         },
     ]
 
-Now, let's format this with the :class:`~torchtune.data.Llama2ChatFormat` class and
-see how it gets tokenized. The Llama2ChatFormat is an example of a **prompt template**,
+Now, let's format this with the :class:`~torchtune.models.llama2.Llama2ChatTemplate` class and
```

Comment:
nit but just while you're in here we can probably safely remove L24-25 about "This tutorial requires a version of torchtune > 0.1.1"